### PR TITLE
💄 Collections - Enhance navigation UI on mobile screens

### LINKED ISSUE #4615: Collections - Fix navigation UI on mobile screens

### PR DESCRIPTION
**Pull request by Jord-Gui**
Fixes #4615

- Ensure padding between collection list page and singular pages are consistent
- Ensure navigation bar doesn't overlap with burger icon on smaller screens for the collections singular pages

image

**Figure: CollectionListPage and CollectionUpdatePage have consistent padding around the body**

image

**Figure: CollectionUpdatePage navigation bar not overlapping with the burger icon**

### CODE REVIEW

File: packages/tinacms/src/admin/pages/CollectionUpdatePage.tsx [hunk 1]
```diff
@@ -148,10 +148,10 @@ const RenderForm = ({
     })
   }, [cms, document, relativePath, collection, mutationInfo])
 
-  const navBreakpoint = 1000
+  const navBreakpoint = 1279
```
Comment (ncn-ssw) [inline packages/tinacms/src/admin/pages/CollectionUpdatePage.tsx:151]: **nitpick**: Where did magic number come from?

File: packages/tinacms/src/admin/pages/CollectionCreatePage.tsx [hunk 2]
```diff
@@ -330,10 +329,10 @@ export const RenderForm = ({
     })
   }, [cms, collection, mutationInfo])
 
-  const navBreakpoint = 1000
+  const navBreakpoint = 1279
```
Comment (ncn-ssw) [inline packages/tinacms/src/admin/pages/CollectionCreatePage.tsx:332]: **nitpick**: Where did magic number come from?

**Review by ncn-ssw — APPROVED** [review]
🦙 Manually tested correct, a couple of nitpicks regarding how breakpoints were decided.

File: packages/tinacms/src/admin/pages/CollectionCreatePage.tsx (hunk 2 above)
Comment (Jord-Gui) [reply packages/tinacms/src/admin/pages/CollectionCreatePage.tsx:332]: See above comment

**Pull request merged**